### DISCUSSION TITLE
GitHub Organization Daily Invite Limit

### DISCUSSION THREAD

**Discussion by james-orr** [community/community#33053] [GitHub Education]
I am a university-level instructor. One course I teach is an introductory large-enrollment CS class. We use Github Classroom to distribute and college the various assignments. Our process is to create a new org each semester to house the template repositories and the students' code.

Unfortunately, at the beginning of the semester, there is always an issue that we run into. During the first day of class when students accept the first assignment, we always run into a wall. After a certain point, no more students can accept the assignment, but the next day a certain amount more can, and the process continues for a week or so until everyone is able.

After some back-and-forth with GitHub support last spring (someone else is teaching the class this semester), I think we determined that there is a limit to the number of people that can join the GiHub Organization on any given day. The only "solution" given by GitHub Support was to increase the daily invite limit for my specific org, but by the time we came to that, all students had successfully joined the org, and the issue was not longer relevant (all students could accept all future assignments since they were already part of the org).

Multiple large classes at my university ran into the same problem at the beginning of this fall semester, so I come here asking if there is anything that we can do to mitigate this problem. Surely we aren't the only ones dealing with it.

**Comment by maankhedr** [comment]
I am in the same boat, luckily when i contacted them they gladly upped my limit to 300/day for the week to accommodate to my needs
also i have read on one thread that if the organization is a month old the limit is pushed up to 500 but i am not sure of this information so you might want to verify that

**Reply by james-orr** [reply]
Unfortunately neither of these is a workable long-term solution. 

We have some intro classes that are larger than 300 or even 500 students, and they all typically join the organization on the first day of class, so I'm just at a slightly smaller version of the same problem.

Even if the limit were to automatically expand to be large enough a month into the organization's life, that isn't very practical. We are often still in the previous semester a month prior to creating the organization for the upcoming semester and are focused exclusively on the semester that is  ending. It wouldn't occur to me when the fall semester is ending that instead of creating my final exam for that semester, I should be creating a GitHub organization for the upcoming semester.

I think a way to even temporarily disable the daily limit is the only workable solution, and it is a large enough issue for our large classes each semester that our department has considered moving away from GitHub Classrooms and to another platform for at least our introductory classes because of this.

**Reply by maankhedr** [reply]
Update:
so i asked the rep regarding the limit information i read about and it is not true
the only way to get the limit up is to contact a rep and explain to them that you need the limit temporarily upped for class registration and they will be happy to comply

**Reply by timdrichards** [reply]
How exactly did you contact a rep? The only way I found to contact them is through an online form to create a ticket.
